scripts/examples/gn_efr32_example.sh

```diff
@@ -220,7 +220,15 @@ else
     BUILD_DIR=$OUTDIR/$SILABS_BOARD
     echo BUILD_DIR="$BUILD_DIR"
     if [ "$USE_WIFI" == true ]; then
-        gn gen --check --fail-on-unused-args --export-compile-commands --root="$ROOT" --dotfile="$ROOT"/build_for_wifi_gnfile.gn --args="silabs_board=\"$SILABS_BOARD\" $optArgs" "$BUILD_DIR"
+        # wifi build
```

Comment:
I believe this should be re-thought

When building a Siwx917 example board is always BRD4325B and it doesn't need the --wifi preset anymore.

it should use the normal build option with the right paths and right boards

Reply:
So I tested on your branch with the latest changes. And indeed we shouldn't need this if are any --wifi preset for siwx917 soc.

currently I can build with
`./scripts/examples/gn_efr32_example.sh examples/lighting-app/silabs/SiWx917/ out/lighting_app_siwx BRD4325B use_SiWx917=true`

And we should not need to pass `use_SiWx917=true` either. If you do need this parameter specific if in the building part we can set use_SiWx917=true  in `silabs_board.gni` for board `BRD4325B` since it is always true for that board :)

Reply:
@jmartinez-silabs true, there is actually no point of having the --wifi SiWx917 flag since it should be enabled by default. We kept this flag just to differ from the thread. --wifi flag confirms that it is a wifi specific hardware and user doesn't get confused with the thread and since we have the ncp solution of SiWx917 as well this tells that the BRD4325B board uses the SiWx917 as the wifi part. 
Would like your opinion on this.

Reply:
Modified on the latest commit, it is. The project is now building with 
`./scripts/examples/gn_efr32_example.sh examples/lighting-app/silabs/SiWx917/ out/SoC/lighting-app/ BRD4325B`